frontend/src/routes/portfolio/+page.svelte

```diff
@@ -1,11 +1,19 @@
+<script>
```

Comment:
There's no type checking on this page - remember to use `lang="ts"` on the script tag.

Reply:
Oops my mistake